devel/cquery/Portfile

```diff
@@ -27,6 +27,8 @@ github.setup       cquery-project cquery 20180302 v
 
 fetch.type      git
 
+revision 1
+
```

Comment:
I'm not sure this is needed since we haven't changed the rev of the base version of cquery.